WISE-Community/drawing-tool
# fix(Tool Buttons): Losing highlight when selecting sub item

## Changes

Tool icon remains highlighted after selecting a sub menu item.

## Test

- Click on the shape tool once to select it. It should become highlighted.
- Click on the shape tool again to open the shape menu
- Click on the square
- The shape tool should still be highlighted

This should also work for the line, text and stamp tools.

Closes #2
